### ISSUE TITLE
Consider different keybinding for navigating between chat code blocks

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#195281]
Users can navigate between notebook output cells with `ctrl+alt+pageUp/pageDown`. 

`f9` is associated with debugging but is currently used for navigating between chat code blocks. 

@roblourens any issue with using the one above instead?

Suggestion by @jooyoungseo

**Comment by meganrogge** [comment]
Have some code blocks, use `ctrlCmd+alt+pageUp/pageDown` to navigate between them

![Image](https://github.com/microsoft/vscode/assets/29464607/96de9764-0337-402f-97c6-dcb25c578d92)

**Comment by mjbvz** [comment]
What about for keyboards without page up / page down? I tried using fn+arrows on Mac but couldn't get it to work

**Comment by meganrogge** [comment]
@mjbvz we rely on users having full keyboards for a lot of important things, like `alt+f1` for example, and I haven't heard that it's an issue.

**Comment by mjbvz** [comment]
I don't think fn keys are an issue but a lot of Mac users are on laptop sized keyboards which won't have pageup/down

Still, I you should be able to hold fn and use up/down arrow for page up/down. That seems to work for me with the suggest widget paging commands, but not with this specific keybinding
